boxsdk/object/item.py

```diff
@@ -159,7 +159,7 @@ def move(self, parent_folder):
         }
         return self.update_info(data)
 
-    def get_shared_link(self, access=None, etag=None, unshared_at=None, allow_download=None, allow_preview=None):
+    def get_shared_link(self, access=None, etag=None, unshared_at=None, password=None, allow_download=None, allow_preview=None):
```

Comment:
I think that the password field should be added as the last parameter, so it doesn't break existing code that might pass all the parameters positionally (this would be a bad idea, but it would also be currently supported).